Ravens Matrices.md

```diff
@@ -0,0 +1,36 @@
+# Summary
+> The series of tasks in this article worked to identify individual contributions to group performance to determine "team players." The Ravens Task in specific measured pattern recognition and spatial reasoning. This contributes to the study of IQ or liquid intelligence. 
+
+# References
+> Main paper: https://www.nber.org/system/files/working_papers/w27071/w27071.pdf
+> > Supporting paper: Raven, J. (2003). Raven’s Advanced Progressive Matrices, Test Books 1 and 2: Pearson Assessment.
+> > > CANNOT FIND ACCESS TO THIS PAPER
+
```

Comment:
I think they are actually test practice booklets, I think I found them on the Pearson site: https://www.pearsonassessments.com/store/usassessments/en/Store/Professional-Assessments/Cognition-%26-Neuro/Non-Verbal-Ability/Raven%E2%80%99s-Advanced-Progressive-Matrices/p/100000414.html

Reply:
Were you able to access them for free?

Reply:
No I think they need to be purchased